plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -1098,11 +1098,6 @@ public static String sortingColumnToString(SortingColumn column)
 
     public static boolean isHiveSystemSchema(String schemaName)
     {
-        if ("information_schema".equals(schemaName)) {
```

Comment:
`isHiveSystemSchema("information_schema")` should still return true, because "information_schema" is indeed a Hive system schema.

we should reword the comment though